### DISCUSSION TITLE
Missing delete branch on PR setting

### DISCUSSION THREAD

**Discussion by VaughanNawiri** [community/community#76752] [Pull Requests]
### Select Topic Area

General

### Body

As per this article https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches I can't find a setting under my repo to do this. I want branches to automatically be deleted after a PR is approved and merged.

**Comment by guspan-tanadi** [comment]
Obviously, there are no public repo found in the [profile](https://github.com/VaughanNawiri).
Please create one or by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) [this repo](https://github.com/desktop/desktop) for example.
Then within the repo follow those [article](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches).

**Reply by VaughanNawiri** [reply]
I'm part of a private organisation. I can see all settings, but there's no Pull Request option in this repo. Is it possible because its not a paid plan that I don't see PR setting? Strange tho, because I see all other settings.
